view/templates/peoplefind.tpl

```diff
@@ -7,6 +7,7 @@
 	</form>
 	<div class="side-link" id="side-match-link"><a href="match" >{{$similar}}</a></div>
 	<div class="side-link" id="side-suggest-link"><a href="suggest" >{{$suggest}}</a></div>
+	<div class="side-link" id="side-directory-link"><a href="directory" >{{$directory}}</a></div>
```

Comment:
This would link to the local directory of the node itself. #3567 is requesting a link to the global directory e.g. dir.friendica.social - but this is configurable for each Friendica node.

You can get the current setting from the DB with `Config::get('system', 'directory') ` in the above php file for the widget and pass it over to the template.